README.md

```diff
@@ -7,6 +7,14 @@ EKS-optimized AMI.
 
 **Check out the AMI's [user guide](doc/USER_GUIDE.md) for more information.**
 
+## Excuting ##
+```shell
+EXPORT AWS_PROFILE=prod
```

Comment:
EXPORT in upper case?

i think you can skip changes to this file for this PR (and open one later for adding the section in another PR)